spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -391,6 +391,29 @@ def over_65_birth_year
     end
   end
 
+  describe 'line 16a taxable interest income' do
+    context 'with no interest reports' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_minimal) }
+      it 'does not set line 16a' do
+        expect(instance.lines[:NJ1040_LINE_16A].value).to eq(nil)
+      end
+    end
+
+    context 'with interest reports, but no interest on government bonds' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_one_dep) }
+      it 'does not set line 16a' do
+        expect(instance.lines[:NJ1040_LINE_16A].value).to eq(nil)
+      end
+    end 
+
+    context 'with interest on government bonds' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_two_deps) }
+      it 'sets line 16a to 300 (fed taxable income minus sum of bond interest)' do
```

Comment:
[dust/pebble] could we add numbers to these calculation descriptions? Helps me track down where they're coming from in the test data. 

ie, `"sets line 16a to $300 (fed taxable income $XXX minus sum of bond interest $XXX)"`